packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -107,35 +108,27 @@ const zoomAuth = (credential: CredentialPayload) => {
       zoomRefreshedTokenSchema
     );
 
-    // TODO: If the new token is invalid, initiate the fallback sequence instead of throwing
-    // Expanding on this we can use server-to-server app and create meeting from admin calcom account
-
-    const oldCredential = await prisma.credential.findUniqueOrThrow({ where: { id: credential.id } });
-    const parsedKey = zoomTokenSchema.safeParse(oldCredential.key);
-    if (!parsedKey.success) {
-      return Promise.reject(new Error("Invalid credentials were saved in the DB"));
-    }
```

Comment:
If the old credential was malformed, we would throw this error early without saving the new tokens in the DB. We are also now parsing the old keys in `isTokenValid`